bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -95,12 +92,6 @@ public void initialize() {
         }
     }
 
-    @Override
-    public void dispose() {
-        super.dispose();
-        stopPoll();
```

Comment:
Removing the `stopPoll` is wrong. Otherwise the poll job is never ended.

Reply:
Apologies, I misunderstood your previous comment, fixed in a1f3facd2f